### PR TITLE
Support for additional metadata coming from OJS

### PR DESCRIPTION
**Pull request by ajnyga**
Support for additional metadata fields, not tested!

See https://github.com/kaschioudi/ojs3-markup/pull/31

### CODE REVIEW

**Comment by axfelix** [comment]
Thanks, this looks great! Should have it tested and merged this week...

File: module/MergeXMLOutputs/src/MergeXMLOutputs/Model/Converter/Merge.php [hunk 1]
```diff
@@ -266,56 +266,119 @@ protected function merge()
...
+			$xml .= '<contrib-id contrib-id-type="orcid">'.$c['orcid'].'</contrib-id>';
+			$xml .= '<email>'.$c['email'].'</email>';
+			$xml .= '<aff>'.$c['affiliation'].'</aff>';	
+			$xml .= '<bio>'.$c['bio'].'</bio>';
+			$xml .= "</contrib>\n";
+		}	
+		$xml .= "</contrib-group>";
+
+		/* Issue details */
+		if (isset($metadata['issue-details']){
```
Comment (axfelix) [inline module/MergeXMLOutputs/src/MergeXMLOutputs/Model/Converter/Merge.php]: missing a close parenthesis here?

**Comment by axfelix** [comment]
This appears to be causing XMP (PDF metadata) generation to fail, need to investigate: https://github.com/pkp/ots/blob/master/module/XmpConversion/src/XmpConversion/Model/Converter/Xmp.php

**Comment by ajnyga** [comment]
@axfelix, it is really hard for me to debug the OTS end so do you mind checking this?

**Comment by axfelix** [comment]
Sure, I can do it.

**Comment by ajnyga** [comment]
Thanks, in the future I will probably try to set up a OTS to our own server, but that will probably take some effort.

**Comment by axfelix** [comment]
That's why I made https://github.com/axfelix/ots-vagrant :)

**Comment by ajnyga** [comment]
Oh nice, thanks!

**Pull request merged**